cves/kernel/CVE-2019-19069.yml

```diff
@@ -219,8 +226,9 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
-  note:
+  name: drivers
+  note: |
+    The vulnerability is specificly in the drivers for the Qualcomm FastRPC driver (drivers/misc/fastrpc.c).
```

Comment:
I would suggest adding a description of what this driver does in the subsystem of drivers.

Reply:
I believe it's for optimizing the efficiency of requests being sent between Qualcomm snapdragon family of processors. I wasn't sure if I should include this so I left it out. But, I'll make sure to add it in the next commit.